test/parallel/test-child-process-spawnsync-kill-signal.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+const cp = require('child_process');
+
+if (process.argv[2] === 'child') {
+  setInterval(() => {}, 1000);
+} else {
+  const SIGUSR2 = process.binding('constants').os.signals.SIGUSR2;
```

Comment:
You could DRY this as `const { SIGUSR2 } = process.binding('constants').os.signals`.  Minor thing though.